src/dropdownListModel.ts

```diff
@@ -5,6 +5,8 @@ import { property } from "./jsonobject";
 import { ListModel } from "./list";
 import { PopupModel } from "./popup";
 import { Question } from "./question";
+import { QuestionDropdownModel } from "./question_dropdown";
+import { QuestionTagboxModel } from "./question_tagbox";
 import { CssClassBuilder } from "./utils/cssClassBuilder";
```

Comment:
QuestionTagboxModel - not uses

this.question instanceof QuestionDropdownModel - replace to 
get isSingleChoiceQuestion() { return  this.question.getType() === "dropdown"; }